### DISCUSSION TITLE
Tests problem

### DISCUSSION THREAD

**Discussion by GeorgeF23** [prasanaworld/puppeteer-screen-recorder#51] [Q&A] [answered]
Hello! I am trying to make a contribution to the project, but I have some problems running the tests.
I ran the tests without making any changes to the code and I encountered the following problems:
1. The tests failed at the lines with 'browser.newPage()' but this was fixable by setting the 'PUPPETEER_EXECUTABLE_PATH' to my local google chrome path
2. The tests failed with the following error:   Error {
    message: `FFmpeg path is missing, ␊
     Set the FFMPEG_PATH env variable`,
  }
I tried setting the FFMPEG_PATH env variable but it still does not work.

Am I doing something wrong?

**Comment by prasanaworld** [comment, marked as answer]
Hi @GeorgeF23, Thanks for your showing interest in contributing to this project. 

To overcome this error, you should export the puppeteer execute path before running the test. 

```sh 
export PUPPETEER_EXECUTABLE_PATH=<path of chromium browser>
``` 
(i.e `export PUPPETEER_EXECUTABLE_PATH=/usr/local/bin/chromium`)

Please let me know if you are still facing any issue.
